packages/cdktf/lib/testing/matchers.ts

```diff
@@ -221,36 +221,49 @@ export function toBeValidTerraform(received: string): AssertionReturn {
       false
     );
   }
-
   try {
     const manifest = JSON.parse(
       fs.readFileSync(path.resolve(received, "manifest.json"), "utf8")
     );
 
     const stacks = Object.entries(manifest.stacks);
 
+    const errors: string[] = [];
+
     stacks.forEach(([name, stack]) => {
       const opts = {
         cwd: path.resolve(received, (stack as any).workingDirectory),
         env: process.env,
         stdio: "pipe",
+        shell: true,
       } as any;
       execSync(`${terraformBinaryName} init`, opts);
-      const out = execSync(`${terraformBinaryName} validate -json`, opts);
-
-      const result = JSON.parse(out.toString());
+      // using spawnSync to avoid error thrown by non 0 exit code
+      const out = spawnSync(`${terraformBinaryName} validate -json`, opts);
+      const result = JSON.parse(out.stdout);
```

Comment:
Would using the existing try block that contains the parse suffice? The subsequent catch block could then throw an Error rather than returning false as either the `terraform init` or `terraform validate` call fails in some manner that's seemingly unrelated to the validity of the stack being checked.

Reply:
Yeah I think rethrowing the error the parse will throw with some nicer explanation would be great here